resume-buddy/src/main/webapp/style.css

```diff
@@ -0,0 +1,52 @@
+body,
+h2,
+h3,
+h4,
+h5,
+h6,
+.w3-wide {
+  font-family: "Roboto", sans-serif;
+}
+ 
+.title {
+  font-family: 'Noto Sans TC', sans-serif;
+  font-size: 100px;
+  align-items: center;
+  text-align: center;
+  }
+ 
+.logo {
+  width: 130px;
+  height: 130px;
+  float: right;
+  margin-right: 50px;
+  margin-top: 10px;
+  margin-bottom: 10px;
+  transform: rotate(15deg);
+}
+ 
+.window { 
+  margin-left:auto;
+  margin-right:auto;
+  max-width:1250px;
+}
```

Comment:
Since this is only used to style the `<body>`, and there is only ever one of those, I think these should be moved into the `body {...}` style block. Then we don't need to give the `body` a class.

Nit: horizontal center can also be achieved with one line using `margin: auto;`